cmd/scheduler-estimator/app/options/options.go

```diff
@@ -55,6 +60,16 @@ type Options struct {
 	// Parallelism defines the amount of parallelism in algorithms for estimating. Must be greater than 0. Defaults to 16.
 	Parallelism int
 	ProfileOpts profileflag.Options
+	// MetricsBindAddress is the TCP address that the server should bind to
+	// for serving prometheus metrics.
+	// It can be set to "0" to disable the metrics serving.
+	// Defaults to ":10351".
+	MetricsBindAddress string
```

Comment:
@zhzhuang-zju should I change the default of metricsBindAddress to 8080 to make it the same as https://github.com/karmada-io/karmada/blob/master/cmd/agent/app/options/options.go#L106-L110?

Reply:
we can do it in the release-1.12.